packages/perseus/src/widgets/radio/choice.module.css

```diff
@@ -0,0 +1,70 @@
+.choice {
+    --perseus-multiple-choice-spacing: 1.6rem;
+    align-items: flex-start;
+    border-bottom: var(--wb-border-width-thin) solid
+        var(--wb-semanticColor-core-border-neutral-subtle);
+    display: flex;
+    font-size: var(--wb-font-heading-size-medium);
+    /* "gap" is being used to put horizontal space between the indicator and the content.
+       That way, we can keep all spacing in this file, and not have to apply styling to the indicator itself.
+    */
+    gap: var(--perseus-multiple-choice-spacing);
+    line-height: var(--wb-font-heading-lineHeight-medium);
+    min-height: calc(var(--perseus-multiple-choice-spacing) * 4);
+    padding: var(--perseus-multiple-choice-spacing);
+    width: 100%;
+}
+
+.choice:hover {
+    cursor: pointer;
+}
+
+/* Show a bottom border on the choice when:
+      - the choice is not in review mode AND
+         - the choice is hovered OR
+         - the next choice is hovered OR
+         - the choice indicator is focused OR
+         - the next choice indicator is focused
+ */
```

Comment:
Makes sense - so we don't show the bottom border when we're in review mode and no choice is hovered or focused?